app/views/businessmatching/updateservice/add/msb_subservices.scala.html

```diff
@@ -45,30 +45,30 @@
       ) {
           @checkbox(
             f = f("msbServices[]"),
-            labelText = "businessmatching.updateservice.msb.services.list.lbl.01",
+            labelText = "businessmatching.updateservice.msb.services.list.lbl.03",
             value = "01"
           )
           @checkbox(
             f = f("msbServices[]"),
-            labelText = "businessmatching.updateservice.msb.services.list.lbl.02",
+            labelText = "businessmatching.updateservice.msb.services.list.lbl.04",
```

Comment:
As above and for the others where we have switched labels.

Reply:
^